src/combobox/ComboBox.ts

```diff
@@ -169,19 +162,19 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 	private _onInputKeyDown(event: KeyboardEvent) {
 		const { results = [] } = this.properties;
 
-		switch (event.keyCode) {
-			case keys.up:
+		switch (event.which) {
```

Comment:
`event.which` normalizes `event.keyCode` and `event.charCode` which doesn't matter much for these keys, but if we're choosing a standard I'd rather go with `which`.

Safari supports `event.key` as of about a week ago, but the Android browser does not. I forget if we support it, but if not I'd be happy to switch to `.key` and drop this entire util.